cli/util/util.go

```diff
@@ -935,3 +936,8 @@ func CopyFileDeep(src, dst string) error {
 	}
 	return copy.Copy(src, dst)
 }
+
+// DirIsWritable checks if the current user has the write access to the passed directory.
+func DirIsWritable(dir string) bool {
```

Comment:
Why do we need this in `enable` functionality?

Reply:
removed